src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +34,15 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables){
+        String resultString = "";
+        for(Printable printable: printables){
+            resultString += printable.getPrintableString();
```

Comment:
Can try out doing this using StringJoiner which makes your code "prettier" 
Read about it [here](https://stackoverflow.com/questions/27522563/why-stringjoiner-when-we-already-have-stringbuilder)